src/main.rs

```diff
@@ -473,9 +478,9 @@ fn main() {
     );
 
```

Comment:
Instead of applying the demangling in each output function, it'd be preferable to apply it to the result first, then pass the demangled result to the output functions

Reply:
Otherwise we have to duplicate the demangling support in all output functions

Reply:
Some outputs don't need to have demangled symbols... so I added this feature only in outputs which outputs function name to avoid useless stuff for other outputs.